recipes/nanopb/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, replace_in_file
+from os import path
+
+class NanoPbConan(ConanFile):
+    name = "nanopb"
+    version = "0.4.8"
+    license = "zlib"
+    url = "https://jpa.kapsi.fi/nanopb/"
+    description = "Protocol Buffers with small code size"
+    settings = "os", "compiler", "build_type", "arch"
+    exports = '*'
```

Comment:
There's no extra files, `conanfile.py` and `conanadata.yml` as handled by default 

```suggestion
```